test/verify/check-login

```diff
@@ -133,6 +133,16 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
             b.open("/system")
             b.wait_not_visible("#option-group")
 
+        if not m.atomic_image:
+            # login with user shell that prints some stdout/err noise
+            # having stdout output in ~/.bashrc confuses docker, so don't run on Atomic
+            m.execute("echo 'echo noise-rc-out; echo noise-rc-err >&2' >> ~admin/.bashrc")
+            m.execute("echo 'echo noise-profile-out; echo noise-profile-err >&2' >> ~admin/.profile")
+            login("admin", "foobar")
+            with b.wait_timeout(10):
```

Comment:
What's the reason to use a faster timeout here?  I think we shouldn't change it.

Reply:
No particular reason really, it's just copy&paste from all the other similar instances in this test. I'm happy to change it wholesale.

Reply:
Pushed!

Reply:
Ah, I see.  Thanks!